ci-operator/config/jws-qe/interop-ocp-ci/jws-qe-interop-ocp-ci-main__ocp-4.14-lp-interop.yaml

```diff
@@ -43,6 +40,7 @@ tests:
               {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
```

Comment:
We can remove the success rules as it is already defined in https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json 

Should be able to remove it from all the configs in this PR